.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Do we need this comment? Looks like it was generated. Check throughout the files for this.

Reply:
We talked about each of these in our meeting so I will just resolve them all, this comment is to make note that they've all been addressed